webp/view/src/main/cpp/webp_decode.cpp

```diff
@@ -124,7 +125,7 @@ void WebpDecoder::DecodeFrameInternal() {
 
     status = WebPDecode(buf, len, &config);
     WebPFreeDecBuffer(&config.output);
-    delete buf;
+    delete [] buf;
```

Comment:
yeah, it is fixed with add back in delete [].  will look to make it real C++ like later thanks